#cmd: 0-start, 1- config success, 2-config failure
int8    cmd
string ssid